external/header.php

```diff
@@ -118,6 +118,12 @@ function () {
         } else {
             $data['profile'] = xhprof_disable();
         }
+        
+        $profile = [];
+        foreach($data['profile'] as $key => $value) {
+            $profile[strtr($key, ['.' => '_'])] = $value;
+        }
+        $data['profile'] = $profile;
```

Comment:
Won't you lose filename information this way?

Reply:
Maybe I should use `save.handler` from config and if that's `== 'mongodb'` the keys are updated?


On xhgui is all as expected, however, instead of seeing `admin/index.php` in the function list and function details, you see `admin/index_php`. Mongo won't accept `dot` as index.

------------

## Before the PR

![Screenshot 2019-08-21 at 17 23 34 578](https://user-images.githubusercontent.com/930792/63449769-a8c1da00-c438-11e9-8366-209c2cffa325.png)

------------

## After the PR

![Screenshot 2019-08-21 at 17 24 00 147](https://user-images.githubusercontent.com/930792/63449779-afe8e800-c438-11e9-846e-4f2282eefdaa.png)